docker-compose.dev.yml

```diff
@@ -50,16 +54,42 @@ services:
       - mongodb
     networks:
       - frontend
+  
+  ### Lambda Function #############################
+  lambda:
+    container_name: ${PROJECT}-lambda
+    image: lambci/lambda:nodejs12.x
+    environment:
+      - DB_SERVER=${DB_SERVER}
+      - DB_PORT=${DB_PORT}
+      - DB_USER=${DB_USER}
+      - DB_PASSWORD=${DB_PASSWORD}
+      - DB_NAME=${DB_NAME}
+    expose:
+      - "9001"
+    ports:
+      - "9001:9001"
+    environment:
+      - AWS_DEFAULT_REGION=ca-central-1
+      - DOCKER_LAMBDA_WATCH=1
+      - DOCKER_LAMBDA_STAY_OPEN=1
```

Comment:
The makefile was loading the env vars properly, the problem was this **environment:** duplicated, really don't know how I let this pass, sorry 🤦 it should be:

    environment:
      - DB_SERVER=${DB_SERVER}
      - DB_PORT=${DB_PORT}
      - DB_USER=${DB_USER}
      - DB_PASSWORD=${DB_PASSWORD}
      - DB_NAME=${DB_NAME}
      - AWS_DEFAULT_REGION=ca-central-1
      - DOCKER_LAMBDA_WATCH=1
      - DOCKER_LAMBDA_STAY_OPEN=1

Reply:
Great catch, fixed!